legal-api/report-templates/template-parts/registration-statement/party.html

```diff
@@ -0,0 +1,70 @@
+<div class="no-page-break">
+  <div class="separator mt-4"></div>
+  <div class="section-title mt-4">
+    {% if business.legalType == 'SP' %}
+      <span>Proprietor</span>
+    {% elif business.legalType == 'GP' %}
+      <span>Partner</span>
+    {% endif %} Information
+  </div>
+  <div class="mt-4">
+    {% if business.legalType == 'SP' %}
+      <div class="section-data">The Proprietor listed below  is the only member of the Sole Proprietorship.</div>
```

Comment:
there is an extra space between `below` and `is` on L12

Reply:
Nice catch!